framework/test/unit/modules/interoperability/store.spec.ts

```diff
@@ -237,7 +246,10 @@ describe('Base interoperability store', () => {
 
 		it('should set appropriate terminated state for chain id in the terminatedState sub store if chain account exists for the id but state root is not provided', async () => {
```

Comment:
missing assertion for emitted event and `await terminatedStateSubstore.set` similar to above comment and also applies for the following test cases